v0/destinations/marketo/util.js

```diff
@@ -126,50 +125,46 @@ const sendPostRequest = async (url, data, options) => {
   return processedResponse;
 };
 
-// eslint-disable-next-line no-unused-vars
-const responseHandler = (destinationResponse, _dest) => {
-  const message = `[Marketo Response Handler] - Request Processed Successfully`;
-  const { status } = destinationResponse;
-  // if the responsee from destination is not a success case build an explicit error
-  if (!isHttpStatusSuccess(status)) {
-    throw new ErrorBuilder()
-      .setStatus(status)
-      .setMessage(
-        `[Marketo Response Handler] - Request failed  with status: ${status}`
-      )
-      .setDestinationResponse(destinationResponse)
-      .isTransformResponseFailure(true)
-      .setStatTags({
-        destination: DESTINATION,
-        stage: TRANSFORMER_METRIC.TRANSFORMER_STAGE.RESPONSE_TRANSFORM,
-        scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.SCOPE,
-        meta: getDynamicMeta(status)
-      })
-      .build();
+class MarketoNetworkHandler extends GenericNetworkHandler {
+  // eslint-disable-next-line class-methods-use-this
+  responseHandler(destinationResponse) {
```

Comment:
How is this different from the responseHandler defined in Ln74?

Reply:
I can see that those are being used for the intermediate axios calls. @utsabc can you share some more insight on this?

Reply:
If they are strictly different, we should avoid name conflicts